src/mbedtls_socket.cc

```diff
@@ -213,18 +213,21 @@ int MbedtlsSocket::Accept4(int /*sockfd*/, sockaddr* /*addr*/, socklen_t* /*addr
   throw std::runtime_error("no crt provided");
 }
 
-int MbedtlsSocket::Accept(int sockfd, const std::string& ca_crt,
+int MbedtlsSocket::Accept(int sockfd, sockaddr* addr, socklen_t* addrlen, int flags, const std::string& ca_crt,
                           const std::string& sever_crt, const std::string& sever_key) {
+  int connection_fd = sock_->Accept4(sockfd, addr, addrlen, flags);
```

Comment:
```suggestion
  const int connection_fd = sock_->Accept4(sockfd, addr, addrlen, flags);
```
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rconst-immutable